src/MinimumNodeService.cpp

```diff
@@ -76,20 +76,9 @@ void MinimumNodeService::setUninitialised()
 //
 /// revert from Normal to Uninitialised mode
 //
-void MinimumNodeService::revertUninitialised()
+void MinimumNodeService::initSetupFromNormal()
 {
   // DEBUG_SERIAL << F("> reverting to Uninitialised mode") << endl;
-  // send NNREL message
-  controller->sendMessageWithNN(OPC_NNREL);
-  setUninitialised();
-}
-
-//
-/// change or re-confirm node number
-//
-
-void MinimumNodeService::renegotiate()
```

Comment:
This function is the same as initSetupFromNormal().
Also the term "renegotiate" is misleading as we removed the pushbutton initiated renegotiation functionality.

Reply:
This function could become initSetupFromNormal().  I kept the original name as referring to renegotiating the node number rather than the original concept of CAN ID.  However, it isn't really renegotiation as we are requesting a node number no negotiating for one.